gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowConfigResourceLocalHandler.java

```diff
@@ -163,13 +163,17 @@ public UpdateResponse updateFlowConfig(FlowId flowId, FlowConfig flowConfig, boo
       throw new FlowConfigLoggedException(HttpStatus.S_400_BAD_REQUEST,
           "flowName and flowGroup cannot be changed in update", null);
     }
+    FlowConfig originalFlowConfig = getFlowConfig(flowId);
     if (isUnscheduleRequest(flowConfig)) {
       // flow config is not changed if it is just a request to un-schedule
-      FlowConfig originalFlowConfig = getFlowConfig(flowId);
+      originalFlowConfig = getFlowConfig(flowId);
       originalFlowConfig.setSchedule(NEVER_RUN_CRON_SCHEDULE);
       flowConfig = originalFlowConfig;
     }
 
+    // Carry forward the requester list property since it is added at time of creation
+    flowConfig.getProperties().put(RequesterService.REQUESTER_LIST, originalFlowConfig.getProperties().get(RequesterService.REQUESTER_LIST));
```

Comment:
if it is an unscheduled request, flowConfig is pointing to originalFLowConfig and this line will make no sense then. so Line 175 should be at L167 (before the if condition). Otherwise requester list will be lost set in unscheduled specs.

Reply:
I don't think it actually matters? They are both pointing to originalflowconfig, so it will just replace requester list with itself, it won't lose it.

Either way moved it.

Reply:
Yea, you are right.